src/ProcessOutput.php

```diff
@@ -80,4 +95,20 @@ public function getLegend()
 
         return sprintf("\nLegend: %s\n", implode(', ', $symbols));
     }
+
+    /**
+     * @param int $maxLineWidth
+     * @param int $currentPositionInLine
+     * @return string
+     */
+    private function getSpaceForSymbols($maxLineWidth, $currentPositionInLine)
+    {
+        $emptySpace = '';
+
+        for ($i = 0; $i < $maxLineWidth - $currentPositionInLine; $i++) {
```

Comment:
http://php.net/manual/en/function.str-repeat.php supported since php4

Reply:
or there exists alsoo this http://php.net/manual/en/function.str-pad.php

Reply:
Changed. Are you already satisfied?

Reply:
I take it like yes